templates/block/block--system-menu-block.html.twig

```diff
@@ -2,33 +2,6 @@
 /**
  * @file
  * Theme override for a menu block.
- *
- * Available variables:
```

Comment:
**Question:** Is there still a way to view these variables if a developer wanted to modify?

Reply:
Yes all variables can be seen on the base template `block.html.twig`